src/ui/qgsnewvectorlayerdialogbase.ui

```diff
@@ -69,22 +69,29 @@
      <property name="title">
       <string>Type</string>
      </property>
-     <layout class="QHBoxLayout" name="horizontalLayout">
-      <item>
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="0" column="1">
+       <widget class="QRadioButton" name="mLineRadioButton">
+        <property name="text">
+         <string>Line</string>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
        <widget class="QRadioButton" name="mPointRadioButton">
         <property name="text">
          <string>Point</string>
         </property>
        </widget>
       </item>
-      <item>
-       <widget class="QRadioButton" name="mLineRadioButton">
+      <item row="1" column="0" colspan="3">
+       <widget class="QCheckBox" name="mGeometryWithZCheckBox">
         <property name="text">
-         <string>Line</string>
+         <string>Geometries with Z coordinate</string>
```

Comment:
Hi, that could be rename from "Geometries with Z coordinate" to "Geometries with Z/M coordinate" since M is also saved apparently